api/package.json

```diff
@@ -44,7 +44,7 @@
     "jsonwebtoken": "^9.0.0",
     "jwks-rsa": "3.1.0",
     "lodash": "^4.17.21",
-    "minio": "^7.1.0",
+    "minio": "^7.1.1",
```

Comment:
Please remove the `^` from minio's version to prevent upgrade during maintenace or lockfile replacement.